packages/appstore/src/components/onboarding-new/__tests__/static-dashboard.spec.tsx

```diff
@@ -0,0 +1,64 @@
+import React from 'react';
+import { render, screen } from '@testing-library/react';
+import StaticDashboard from '../static-dashboard';
+import { StoreProvider } from '@deriv/stores';
+
+describe('<StaticDashboard />', () => {
+    let mockRootStore: any;
+    beforeEach(() => {
+        mockRootStore = {
+            client: {
+                is_eu_country: false,
+                is_logged_in: true,
+            },
+            traders_hub: {
+                content_flag: 'something',
+            },
+        };
+    });
```

Comment:
```suggestion
        const mock_root_store = {
            client: {
                is_eu_country: false,
                is_logged_in: true,
            },
            traders_hub: {
                content_flag: 'something',
            },
        };
    });
```

Do we need the beforeEach function ?

Reply:
After you make suggested changes run the command
`npm run test:jest static-dashboard.spec.tsx`
to generate the test coverage